ipn/localapi/localapi.go

```diff
@@ -2707,6 +2731,44 @@ func (h *Handler) serveUpdateProgress(w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(ups)
 }
 
+// serveDNSOSConfig serves the current system DNS configuration as a JSON object, if
+// supported by the OS.
+func (h *Handler) serveDNSOSConfig(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "GET" {
```

Comment:
Could use `httpm.GET` here instead? Up to you though

Reply:
Done